autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/feature/plugin/AutofillRemoteFeature.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.autofill.impl.feature.plugin
+
+import com.duckduckgo.anvil.annotations.ContributesRemoteFeature
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.feature.toggles.api.features.AutofillFeature
+
+@ContributesRemoteFeature(
+    scope = AppScope::class,
+    boundType = AutofillFeature::class,
+    toggleStore = AutofillFeatureToggleStore::class,
+    settingsStore = AutofillFeatureSettingStore::class,
+    exceptionsStore = AutofillFeatureExceptionStore::class,
+)
+interface AutofillRemoteFeature
```

Comment:
It's not clear to me, but I don't think this interface is used externally for anything. Can we make that more obvious that's it a dummy/placeholder (if it is indeed just that) by:

- marking it as `private`
- calling it something like `Unused` or something
- updating the example in `ContributesRemoteFeature`

or maybe i'm missing something about it and it is important? 

Related to this, is it not possible to ditch this interface altogether and move all of the properties straight onto the actual feature interface? i.e., instead of defining them all here like this, could this `@ContributesRemoteFeature` annotation replace `@FeatureName("autofill")`?

Reply:
That's now renamed to `private interface UnusedAutofillRemoteFeatureCodegenTrigger`